src/controllers/notifyControllers.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable require-jsdoc */
+import models from '../models';
+
+const { notifications } = models;
+
+export default {
+
+  getAllNotification: async (req, res) => {
+    try {
+      const user_id = req.users.id;
+      const allRequests = await notifications.findAll({ where: { receiver: user_id } });
+      return res.status(200).json({ status: 200, data: allRequests });
+    } catch (error) {
+      res.status(500).json({ status: 500, error });
+    }
+  },
+
+  readAllNotification: async (req, res) => {
+    try {
```

Comment:
@ChidiChuks  You really put in a lot of efforts into this

Are you supposed to implement this function **readAllNotification**? because there is a task on the PT that has to do with reading notifications

Reply:
> @ChidiChuks You really put in a lot of efforts into this
> 
> Are you supposed to implement this function **readAllNotification**? because there is a task on the PT that has to do with reading notifications

Oh yes, I had to do this to be able to test the algorithm and see that the notification is working